pi-gpiod: improved/fixed EN localisation and added DE help

heikokue opened the pull request:
<!--
## Before you hit that Submit button....

Please read our [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
before submitting a pull-request.

## Types of changes

What types of changes does your code introduce?
Put an `x` in the boxes that apply
-->

- [?] Bugfix (non-breaking change which fixes an issue)
- [?] New feature (non-breaking change which adds functionality)

<!--
If you want to raise a pull-request with a new feature, or a refactoring
of existing code, it **may well get rejected** if it hasn't been discussed on
the [forum](https://discourse.nodered.org) or
[slack team](https://nodered.org/slack) first.

-->

## Proposed changes
the headline says all
<!-- Describe the nature of this change. What problem does it address? -->

## Checklist
<!-- Put an `x` in the boxes that apply -->

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red-nodes/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [ ] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

heikokue commented:
I've no more open things to do regarding this node.
So it can be merged and bumped

dceejay commented on hardware/pigpiod/locales/en-US/pi-gpiod.json:
```diff
@@ -34,7 +34,7 @@
             "in": "Only Digital Input is supported - input must be 0 or 1.",
             "dig": "Digital output - input must be 0 or 1.",
             "pwm": "PWM output - input must be between 0 to 100.",
-            "ser": "Servo output - input must be between 0 to 100. 50 is centre.<br/>min. must be 500 �s or more, max. must be 2500 �s or less.",
```
Why change these ? micro seconds is correct symbol...

heikokue commented:
We added a wrong micro-symbol µ with PR #764
No I see: 
![grafik](https://user-images.githubusercontent.com/74769863/112146615-15d0bf00-8bdc-11eb-86fb-cfdf0e1c2823.png)
What is the correct file coding?
I assumed UTF-8, but at the merge the µ was corrupted.
So I changed here µ to u to get on the safe side.
Or what do you prefer?
